src/CMakeLists.txt

```diff
@@ -23,17 +23,18 @@ set(CGI_SRC
 )
 
 # create binary
-add_library(${PROJECT_NAME}-shared SHARED ${CGI_SRC})
-set_target_properties(${PROJECT_NAME}-shared PROPERTIES
+add_library(${PROJECT_NAME} ${CGI_SRC})
+set_target_properties(${PROJECT_NAME} PROPERTIES
 	OUTPUT_NAME	${PROJECT_NAME}
 	SOVERSION	${PROJECT_VERSION_MAJOR}
 	VERSION		${PROJECT_VERSION}
 )
 
 # install binary
-install(TARGETS ${PROJECT_NAME}-shared
+install(TARGETS ${PROJECT_NAME}
 	EXPORT ${PROJECT_NAME}-targets
 	LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+	ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
```

Comment:
How is that related to the BUILD_SHARED_LIBS variable change?

Reply:
If BUILD_SHARED_LIBS is off, a static library is built, and cmake uses the "ARCHIVE DESTINATION" to know where to install it, while the "LIBRARY DESTINATION" is used when a shared library is built.